README.md

```diff
@@ -19,6 +24,11 @@ public class Status {
     private final static Map<String, Status> values = new HashMap<String, Status>();
     public final static Status OPEN = Status.fromString("OPEN");
     public final static Status CLOSED = Status.fromString("CLOSED");
+    /**
+     * Set containing all enum values declared at compile time.use it in your application to iterate over declared values.
+     *
+     */
+    public final static Set<Status> declaredValues = new HashSet<Status>(Arrays.asList(Status.OPEN, Status.CLOSED));
```

Comment:
We should wrap it in `Collections.unmodifiableSet(...)` to prevent accidental modifications.

Reply:
good point! should we also declare it as _UnmodifiableSet_ instead of _Set_ ?

Reply:
Actually, the method returns `Set`, so there is no `UnmodifiableSet` type visible here.